### PR TITLE
feat: Adding the Terraform module

### PR DESCRIPTION
**Pull request by gatici**
# Description

This PR adds the Terraform module for the charm. Hence, the charm could be deployed using Terraform.

# Checklist:

- [x] My code follows the [style guidelines](/CONTRIBUTING.md) of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [ ] I have added tests that validate the behaviour of the software
- [ ] I validated that new and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have bumped the version of the library

### CODE REVIEW

File: terraform/LICENSE
Comment (gruyaume) [inline terraform/LICENSE:1]: Do we need another License file? we already have one in the root directory

File: terraform/CONTRIBUTING.md
Comment (gruyaume) [inline terraform/CONTRIBUTING.md:1]: The README.md and CONTRIBUTING.md have very similar content. Do we need the two?

File: terraform/LICENSE
Comment (gatici) [reply terraform/LICENSE:1]: It seems we can remove it.

Comment (gatici) [reply terraform/LICENSE:1]: Done

**Review by Gmerold — APPROVED** [review]

**Pull request merged**